### PR TITLE
Update to fix timer

### PR DESCRIPTION
**Pull request by mykelp**
#6

### CODE REVIEW

File: app/components/Auction.js [hunk 1]
```diff
@@ -167,6 +168,15 @@ export default class Auction extends Component {
     else return colors.blue;
   }
 
+  @computed
+  get endDate() {
```
Comment (mertcelebi) [inline app/components/Auction.js]: Hmm, the `Countdown` would get the auction do this calculation itself to not duplicate the logic. But now it's fine.

Comment (mertcelebi) [reply app/components/Auction.js]: But thoughts on this?

**Review by mertcelebi — APPROVED** [review]
Looks good

File: app/components/Auction.js (hunk 1 above)
Comment (mykelp) [reply app/components/Auction.js]: Ideally would like to create a helper method that could be shared between the two components.  Countdown doesn't really need to know about an auction. Just needs to know about times.

File: app/utils.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import moment from "moment";
```
Comment (mertcelebi) [inline app/utils.js:1]: Cool

File: app/components/Home.js [hunk 3]
```diff
@@ -158,7 +164,7 @@ export default class Home extends Component {
               }
             </Description>
             <Divider padded={1.5} />
-            <CountDown endDate={new Date().getTime() + 100000000} />
+            <CountDown endDate={this.endDate(auctionOfInterest)} />
```
Comment (mertcelebi) [inline app/components/Home.js]: This and `Auction.js` should use the `utils` one. And remove the duplicate code

**Review by mertcelebi — CHANGES_REQUESTED** [review]
Couple changes

File: app/components/Home.js (hunk 3 above)
Comment (mykelp) [reply app/components/Home.js]: It does use the `utils` one. It just calls the local function which redirects to the `utils` function with the required inputs.

Comment (mertcelebi) [reply app/components/Home.js]: Hmm, that seems kinda weird. This still duplicates the code (and actually leads to more code probably). Maybe `getEndDate` should accept an auction then?

Comment (mykelp) [reply app/components/Home.js]: Don't necessarily agree. Computation is shared logic, but supplying data to the computation is different.  I changed anyway to do it a bit differently.

**Review by mertcelebi — APPROVED** [review]
Looks good

**Pull request merged**